GoogleForCreators/web-stories-wp
# Code Quality: Remove `mediaRecording` feature flag code

## Context

This feature has been enabled by default in v1.23.0. 



## Summary

Removes mediaRecording feature flag



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Ensure Video Optimization via Settings is turned on
2. Create a story 
3. Note the Record Video / Audio button is available by default
4. Record video + Insert video
5. Remove the video 
6.  With an `empty canvas` use the right click menu to select `Record Video`


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #11998
